### PR TITLE
feat(radiogroup): add RadioGroup component

### PR DESCRIPTION
**Pull request by robertmihaiman**

### CODE REVIEW

File: src/components/RadioGroup/RadioGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { States } from 'storybook-states';
+import { RadioButton } from '../RadioButton';
+import { RadioGroup } from './RadioGroup';
+import { Text } from '../Text';
+import { Stack } from '../Stack';
+
+export default { title: 'Components / RadioGroup' };
+
+export const Default = () => (
+  <States>
```
Comment (hevele-moda) [inline src/components/RadioGroup/RadioGroup.stories.tsx:12]: I think `States` is unnecessary here.

**Review by hevele-moda — APPROVED** [review]

File: src/components/RadioGroup/RadioGroup.stories.tsx (hunk 1 above)
Comment (robertmihaiman) [reply src/components/RadioGroup/RadioGroup.stories.tsx:12]: it displays that wrapper, makes things look consistent

Comment (robertmihaiman) [reply src/components/RadioGroup/RadioGroup.stories.tsx:12]: this:
Screenshot 2020-10-23 at 13 30 14

**Review by hevele-moda — APPROVED** [review]

**Pull request merged**